### ISSUE TITLE
Translating the guides to other languages

### ISSUE THREAD

**Issue by MatheusRich** [luckyframework/website#234]
I know both Crystal and Lucky are not 1.0 yet, so how are you thoughts in introducing some translated versions of the docs? I'm a native portuguese speaker, so I could help on that.

**Comment by jwoertink** [comment]
It would be neat, but my only fear is having to maintain that. We're constantly updating the docs, and since I only know 2 languages (English, and Bad English), keeping any docs in another language becomes really difficult. I'd have to call you up every time we made an update 😅 

I think the other issue is, that would put translations in to a YAML config file, and with the docs being written in markdown, it would get real complicated. Like take this example

```
## Query Objects
    When you [generate a model](),
    Avram will create a Query class for you in `./src/queries/{model_name}_query.cr`.
    This class will inherit from `{ModelName}::BaseQuery`. (e.g. with a `User` model you get
    a `User::BaseQuery` class).
    
   CODE BLOCK
    # src/queries/user_query.cr
    class UserQuery < User::BaseQuery
    end
    
```

We would need a translation for this one title, and then breaking out where we have translations for main words, but none for the code examples (unless we also translated code examples? ).

Maybe there's a free service out there where we could just pass a query param, and it just does it for us like google translate or something?

**Comment by MatheusRich** [comment]
Yeah, that's what I thought. Maybe we'll have to wait 'till 1.0 to start translating it. I'll be happy to help on the pt-br translation when we get there!

**Comment by jwoertink** [comment]
Yeah, that'd be amazing! We appreciate the help ❤️

**Comment by Noctilucente** [comment]
I'm a native portuguese speaker too and I would be glad to help with this.
